OracleAnalytics/dockerfiles/6.4/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2022 Oracle and/or its affiliates.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Analytics Server 2022 (6.4)
+# 
+# REQUIRED FILE(S) TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) Oracle_Analytics_Server_Linux_2022(6.4).zip
+#     Download the installer file from https://www-sites.oracle.com/solutions/business-analytics/analytics-server/analytics-server.html
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run: 
+#      $ docker build -t oracle/analyticsserver:6.4 .
+#
+# Pull base image
+# ---------------
+FROM oracle/fmw-infrastructure:12.2.1.4
+
+# Maintainer
+# ----------
+MAINTAINER Padam Bengani <padam.bengani@oracle.com>
```

Comment:
The `MAINTAINER` tag is deprecated. Please replace this with the required `LABEL` directives as documented in the [contribution guide](https://github.com/oracle/docker-images/blob/main/CONTRIBUTING.md).